lib/sycamore/sycamore/connectors/weaviate/weaviate_reader.py

```diff
@@ -92,6 +92,9 @@ def to_docs(self, query_params: "BaseDBReader.QueryParams") -> list[Document]:
                 | unflatten_data(dict(object.properties), "__")
                 | {"doc_id": str(object.uuid)}
             )  # type: ignore
+            doc.properties["parent_id"] = str(doc.properties.get("parent_id", "")).replace("-", "")
+            doc.properties["element_id"] = str(doc.properties.get("element_id", "")).replace("-", "")
+            doc.parent_id = str(doc.get("parent_id", ""))
```

Comment:
maybe replace with underscore for reconstructability? Also should the doc.parent_id have a replace on it?

Reply:
oh wait this is in the reader? Now I'm def confused how these things got here

Reply:
Discussed offline.